project/AkkaDisciplinePlugin.scala

```diff
@@ -55,7 +55,7 @@ object AkkaDisciplinePlugin extends AutoPlugin with ScalafixSupport {
     silencerSettings ++
     scoverageSettings ++ Seq(
       Compile / scalacOptions ++= (
-          if (!scalaVersion.value.startsWith("2.11") && !nonFatalWarningsFor(name.value)) Seq("-Xfatal-warnings")
+          if (!scalaVersion.value.startsWith("2.11") && !scalaVersion.value.startsWith("2.13") && !nonFatalWarningsFor(name.value)) Seq("-Xfatal-warnings")
```

Comment:
(disable fatal warnings for 2.13 for 2.5, but to be enabled again when we retarget this PR for master)